.github/workflows/ci.yaml

```diff
@@ -103,6 +104,14 @@ jobs:
           filters: |
             openapi:
               - 'openapi/*.yaml'
+      - name: Show initial max_connections value
```

Comment:
We should cleanup this before merging the PR right?

Reply:
yep i'll remove this.